src/styles/clock.scss

```
@@ -0,0 +1,53 @@
+.clock {
+  position: relative;
+  height: $clock-size;
+  width: $clock-size;
+  margin: 50px;
+  background: radial-gradient(#505050, #303030);
+  border: 3px solid #302e2c;
+  box-shadow: 5px silver;
+
+  border-radius: 50%;
```

it may be better to divide different logic areas with spaces
```suggestion
.clock {
  position: relative;
  
  height: $clock-size;
  width: $clock-size;
  margin: 50px;
  
  background: radial-gradient(#505050, #303030);
  box-shadow: 5px silver;
  border: 3px solid #302e2c;
  border-radius: 50%;
```